app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -0,0 +1,134 @@
+$image-overlay-form-width: 400px;
+$image-overlay-label-width: 25%;
+$image-overlay-field-width: 75%;
+
+.alchemy-image-overlay {
+
+  &.open {
+    background-color: rgba(0, 0, 0, 0.6);
+  }
+}
+
+.alchemy-image-overlay-container,
+.alchemy-image-overlay-dialog {
+  width: 100%;
+  height: 100%;
+  max-height: 100%;
+  max-width: 100%;
+}
+
+.alchemy-image-overlay-body {
+  width: 100%;
+  height: 100%;
+}
+
+.alchemy-image-overlay-close {
+  width: 32px;
+  height: 32px;
+  top: 2*$default-padding;
+  right: $image-overlay-form-width + 2*$default-padding;
+  cursor: pointer;
+
+  .icon.close.small {
+    font-size: 32px;
+    color: white;
+  }
+}
+
+.picture-details-overlay {
+  @include box-sizing(border-box);
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: $image-overlay-form-width;
+  height: 100%;
+  background-color: $medium-gray;
+  padding: 2*$default-padding;
+  overflow: auto;
+  @include box-shadow(-2px 0 $default-padding -2px $text-color);
+
+  form .control-label,
+  .resource_info .value label {
+    width: $image-overlay-label-width;
+  }
+
+  form .input .hint {
+    margin-left: $image-overlay-label-width;
+  }
+
+  form .input .select2-container,
+  form .input input[type="text"],
+  .resource_info .value p {
+    width: $image-overlay-field-width;
+  }
+}
+
+#zoomed_picture_background {
+  @include box-sizing(border-box);
+  width: 100%;
```

Comment:
Properties should be ordered cursor, height, margin, padding-left, padding-right, text-align, width